server/src/controllers/__tests__/map.spec.ts

```diff
@@ -51,7 +51,9 @@ describe("MapController", () => {
 
       const fullMap: FullMap | null = await MapController.getMap(new Date())
 
-      expect(database.Map.findOne).toHaveBeenCalledWith({ where: { day: mockMap.day } })
+      expect(database.Map.findOne).toHaveBeenCalledWith({
+        where: { day: expect.anything() },
+      })
```

Comment:
I think this might be a better solution, can you try this and see if it passes the tests.

```JAVASCRIPT
const today = new Date()
const date = new Date(today.getFullYear(), today.getMonth(), today.getDate())
...
const fullMap: FullMap | null = await MapController.getMap(date)
```